app/products/calls/components/calls_custom_message/calls_custom_message.tsx

```diff
@@ -133,97 +138,108 @@ export const CallsCustomMessage = ({
         leaveAndJoinWithAlert(intl, serverUrl, post.channelId, leaveChannelName || '', joinChannelName || '', confirmToJoin, false, Boolean(joinChannelIsDMorGM));
     };
 
+    const title = post.props.title &&
+        <Text style={style.title}>
+            {post.props.title}
+        </Text>;
+
     if (post.props.end_at) {
         return (
-            <View style={style.messageStyle}>
-                <CompassIcon
-                    name='phone-hangup'
-                    size={16}
-                    style={style.phoneHangupIcon}
-                />
-                <View style={style.messageText}>
-                    <FormattedText
-                        id={'mobile.calls_call_ended'}
-                        defaultMessage={'Call ended'}
-                        style={style.startedText}
+            <>
+                {title}
```

Comment:
Sometimes react native gets a bit picky on what you pass in these cases (passing stuff that is not a boolean for example). Have you tested this properly works when the title is empty? If it works well, no changes needed :smile:

Reply:
It works, but I changed it to be safer. 👍